docs/tutorial/workers_workpools.md

```diff
@@ -0,0 +1,168 @@
+
+A deployed flow gets the following additional cababilities:
+
+- flows triggered by scheduling
+- remote execution of flows triggered from the UI
+- flow triggered by automations or events
+
+## What is a Deployment
+
+Deploying your flows is, in essence, the act of informing the Prefect API of:
+
+1. Where to run your flows 
+2. How to run your flows
+3. When to run your flows 
+
+This information is encapsulated and sent to Prefect as a [Deployment](/concepts/deployments/) which becomes a server side object containing the crucial metadata needed for Prefect’s API to execute your flow as desired. Deployments elevate workflows from functions that you call manually to API-managed entities.
+
+Attributes of a deployment include (but are not limited to): 
+
+- Flow entrypoint: path to your flow function would start the flow
+- Workpool: points to the infra you want your flow to run in
+- Schedule: optional schedule for this deployment
+- Tags: optional metadata
+
+Before you build your first deployment, its helpful to understand how Prefect configures flow run infrastrucuture. In order to run **your flows** on **your infrastructure,** you are going to need to set up a work pool and a worker. 
+
+## Why Workpools and Workers
+Easily running Prefect flows locally is great for testing and development purposes, but in a production setting, Prefect provides you with the worker/work pool concepts to allow you to run flows in the environments best suited to their execution. For example, its considered best practice to leverage managed execution platforms, like Kubernetes services or serverless computing environments such as AWS ECS, Azure Container Instances, or GCP Cloud Run. For more information see the [next steps](/tutorials/next_steps/)
+
+## Why Workpools and Workers
+Easily running Prefect flows locally is great for testing and development purposes. But for production settings, Prefect provides you with the worker/work pool concepts to allow you to run flows in the environments best suited to their execution. For example, its considered best practice to leverage managed execution platforms, like Kubernetes services or serverless computing environments such as AWS ECS, Azure Container Instances, or GCP Cloud Run. For more information see the [next steps](/tutorials/next_steps/)
+
+
+Workers and work pools bridge the Prefect orchestration API with your execution environments in your cloud provider.
+
+You can configure work pools on Prefect’s side. They describe the infrastructure configuration for deployed flow runs that get sent to that pool. organize the flows for your worker to pick up and execute. They prioritize the flows and respond to polling from its worker.
```

Comment:
```suggestion
You can configure work pools within the Prefect UI or server. They prioritize the flows and respond to polling from the worker.
```

Reply:
In general, I think we need to draw a clearer distinction between Prefect concepts where _work gets executed_, and Prefect concepts that _organize work_. I've tried to do that in the edit, but please feel free to further edit. That was the intent.